docs/sandbox/DataOverlay.md

```diff
@@ -0,0 +1,66 @@
+# Data Overlay
+TODO DataOverlay
+
+
+## Contact Info
+
```

Comment:
If we need to contact you (future changes), then add enough info here. If you do not want to add your email name and organisation is good enough.

Reply:
You may also include some text here to give us a hint if you would like to be responsible for all changes to DataOverlay or welcome any community updates. We might change it, even if you are in charge, but we will try to contact you first, then.